pkg/executables/tink.go

```diff
@@ -37,3 +40,26 @@ func (t *Tink) PushHardware(ctx context.Context, hardware []byte) error {
 	}
 	return nil
 }
+
+func (t *Tink) GetHardware(ctx context.Context) ([]*hardware.Hardware, error) {
+	params := []string{"hardware", "get", "--format", "json"}
+	data, err := t.Command(ctx, params...).WithEnvVars(t.envMap).Run()
+	if err != nil {
+		return nil, fmt.Errorf("error getting hardware list: %v", err)
+	}
+	var hardwareList []*hardware.Hardware
+	hardwareString := data.String()
+
+	if len(hardwareString) > 0 {
+		hardwareListData := map[string][]*hardware.Hardware{}
+
+		if err = json.Unmarshal([]byte(hardwareString), &hardwareListData); err != nil {
+			return nil, fmt.Errorf("error unmarshling json: %v", err)
```

Comment:
would be good to be more specific about the error and say "error unmarshalling hardware json..."